docker/rust-all.Dockerfile

```diff
@@ -33,7 +33,10 @@ FROM debian-base AS validator
 RUN apt-get update && apt-get install -y libssl1.1 ca-certificates && apt-get clean && rm -r /var/lib/apt/lists/*
 
 ### Needed to run debugging tools like perf
-RUN apt-get update && apt-get install -y linux-tools-4.19 sudo procps
+RUN apt-get update && apt-get install -y linux-perf sudo procps
```

Comment:
Have you tested this out BTW, last time I tried this, I was running into some weird invalid package issue.

Reply:
Yes I did test and the meta package still installs 4.19 but its more resilient, the ln -sf handles this gracefully for us